models/Profile.js

```diff
@@ -31,6 +31,7 @@ const ProfileSchema = new Schema(
       type: Boolean,
       default: true,
     },
+    showStats: Boolean,
```

Comment:
I think the naming might be more consistent if we used something like `isStatsPublic` - sorry to only mention this now